sys/shell/shell.c

```diff
@@ -330,7 +347,16 @@ __attribute__((weak)) void shell_post_command_hook(int ret, int argc,
     (void)argc;
 }
 
-static inline void print_prompt(void)
+/* needed externally by module shell_lock */
```

Comment:
is this still true? I don't see it being called.

Reply:
I believe you meant `flush_if_needed` and not `print_prompt`? `flush_if_needed` is indeed not necessary anymore.